From a1ab5d3be1ccfe850a4420014126372f648e6f65 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Fri, 5 Feb 2016 18:57:17 +0000
Subject: [PATCH 200/462] MANTIS 0037350 initial: integration of EZ-Access
 device for KIS kiosk

added modules
 Massai\cpp\Keyboard\KeyboardDriverService
 Massai\cpp\Keyboard\Keyboard
 Massai\cpp\Keyboard\KeyboardDriverInstaller
 Massai\cpp\Keyboard\KeyboardTest


git-svn-id: svn://localhost/SelfServiceBuild/trunk@306 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/cpp/makefile.mak | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/SelfServiceBuild/Build/cpp/makefile.mak b/SelfServiceBuild/Build/cpp/makefile.mak
index 5c934a3..061c293 100644
--- a/SelfServiceBuild/Build/cpp/makefile.mak
+++ b/SelfServiceBuild/Build/cpp/makefile.mak
@@ -149,7 +149,11 @@ PLATFORM_DIRECTORIES= \
  Massai\cpp\BillingService \
  Massai\cpp\BillingService\static \
  Massai\cpp\specialElectronics \
- Massai\cpp\Illumination
+ Massai\cpp\Illumination \
+ Massai\cpp\Keyboard\KeyboardDriverService \
+ Massai\cpp\Keyboard\Keyboard \
+ Massai\cpp\Keyboard\KeyboardDriverInstaller \
+ Massai\cpp\Keyboard\KeyboardTest \
 
 PLATFORM_INSTALLER_DIRECTORIES= \
  $(INSTALLER_DIRECTORIES)
-- 
2.41.0.windows.1

